### PR TITLE
Updating Loose Files and PSTs folder structure

### PR DESCRIPTION
**Pull request by marstev**
We received an escalation a few months back where customer performed an export, and an additional folder was in the Exchange folder with name <username>_looseFiles. This was not documented anywhere, and customer did not know why they were getting this folder.

### CODE REVIEW

File: microsoft-365/compliance/export-documents-from-review-set.md [hunk 1]
```diff
@@ -89,6 +89,11 @@ If you select this export option, the exported content is organized in the follo
   - Warnings and errors x of z.csv: This file includes information about errors encountered when trying to export from the review set.
   
   - Exchange: This folder contains all content from Exchange stored in PST files. Redacted PDF files cannot be included with this option. If an attachment is selected in the review set, the parent email will be exported with the attachment attached.
+      - This folder could also contain: 
+        - Any messages that were IRM protected and have been decoded.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/export-documents-from-review-set.md]:
```suggestion
        - Any Information Rights Management (IRM) protected messages that have been decoded. 
```

File: microsoft-365/compliance/export-documents-from-review-set.md [hunk 2]
```diff
@@ -89,6 +89,11 @@ If you select this export option, the exported content is organized in the follo
   - Warnings and errors x of z.csv: This file includes information about errors encountered when trying to export from the review set.
   
   - Exchange: This folder contains all content from Exchange stored in PST files. Redacted PDF files cannot be included with this option. If an attachment is selected in the review set, the parent email will be exported with the attachment attached.
+      - This folder could also contain: 
+        - Any messages that were IRM protected and have been decoded. 
+        - Any messages that were error-remediated. 
+        - Any modern attachments/links referenced in e-mails.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/export-documents-from-review-set.md]:
```suggestion
        - Any error-remediated messages. 
        - Any modern attachments or links referenced in emails. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @marstev, kindly commit the minor suggested changes. Many thanks.

**Comment by markjjo** [comment]
@marstev Thanks for submitting this PR. Can you please update the PR and clarify and add more information about this statement: " Encrypted items are not added back into the PST."  Do you mean that encrypted items that are not included in the PST will be exported to the Exchange folder?

**Comment by nahz3matics** [comment]
@markjjo To my understanding the encrypted items will be in the loose files and not actual PST. I can go ahead and remove it for now as the previous lines should be enough detail to what could be included in the loose files.

**Comment by marstev** [comment]
@markjjo Was not signed in with FTE account in my response. I can go ahead and remove it for now as the previous lines should be enough detail to what could be included in the loose files.

**Review by marstev — COMMENTED** [review]
removing line

**Comment by markjjo** [comment]
@marstev Thanks very much for the response and removing the line. I will merge the PR. The updated changes should be live by end of the day.

**Pull request merged**